src/Surfnet/ServiceProviderDashboard/Domain/ValueObject/CreateRoleResponse.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * Copyright 2017 SURFnet B.V.
```

Comment:
```suggestion
 * Copyright 2024 SURFnet B.V.
```

Reply:
This [can|should] be fixed in other files

Reply:
How does this work? Should it always be the current year? Or the year of writing?

Reply:
After discussion, changed!
Is handled by phpstorm from now on. New files will get the correct licence automatically.